include/avif/avif.h

```diff
@@ -799,6 +799,7 @@ typedef struct avifImage
     //
     // To encode any of these boxes, set the values in the associated box, then enable the flag in
     // transformFlags. On decode, only honor the values in boxes with the associated transform flag set.
+    // These also appy to gainMap->image, if any.
```

Comment:
Misspelling: appy => apply

Should we add "gainMap->image->transformFlags is always AVIF_TRANSFORM_NONE" to this comment?